core/src/main/java/org/infinispan/configuration/global/GlobalTracingConfiguration.java

```diff
@@ -0,0 +1,89 @@
+package org.infinispan.configuration.global;
+
+import java.util.Objects;
+
+import org.infinispan.commons.configuration.attributes.AttributeDefinition;
+import org.infinispan.commons.configuration.attributes.AttributeSet;
+
+/**
+ * Configuration for metrics. See {@link GlobalTracingConfigurationBuilder}.
+ */
+public class GlobalTracingConfiguration {
+
+   public static final AttributeDefinition<String> COLLECTOR_ENDPOINT = AttributeDefinition.builder(org.infinispan.configuration.parsing.Attribute.COLLECTOR_ENDPOINT, null, String.class).immutable().build();
+   public static final AttributeDefinition<Boolean> ENABLED = AttributeDefinition.builder(org.infinispan.configuration.parsing.Attribute.ENABLED, true, Boolean.class).immutable().build();
```

Comment:
I think I commented in some other place, but the global attributes cannot be changed at runtime. It seems a useless attribute to have.

Reply:
Yes even if it is sufficient to do not fill the collector endpoint to disable the tracing, I would like to keep it for psychological reason.